fearless/Modules/Staking/StakingMain/StakingMainViewFactory.swift

```diff
@@ -79,15 +79,15 @@ final class StakingMainViewFactory: StakingMainViewFactoryProtocol {
             runtimeService: runtimeService
         )
 
-        let repository: CoreDataRepository<AccountItem, CDAccountItem> =
-            UserDataStorageFacade.shared.createRepository()
-
-        let accountRepositoryFactory = AccountRepositoryFactory(
+        // TODO: check why do we need both factory and repository
```

Comment:
It builds well without `accountRepository`, so it looks like we don't need it

Reply:
Ok, we will try to get rid of it when we integrate shared state